packages/pwa-kit-create-app/package.json

```diff
@@ -6,7 +6,11 @@
   "license": "See license in LICENSE",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mobify/mobify-platform-sdks.git"
+    "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+  },
+  "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/#readme",
```

Comment:
@adamraya – for both of these, would it be valuable to link directly to the specific package?

eg. https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/pwa-kit-create-app#readme

Reply:
That makes sense. I updated the homepage URLs with the specific readmes in both packages.